serverless/src/submitCatalogRestOrder/__tests__/handler.test.js

```diff
@@ -446,7 +446,7 @@ describe('submitCatalogRestOrder', () => {
       '959220857ddbb3b2398ac31a58765df6', // File_hash
       'Limited-MockFile.geojson', // Filename
       1084815579, // Parent_shapefile_id
-      ['1'], // SelectedFeatures
+      JSON.stringify(['1']), // SelectedFeatures
```

Comment:
Should these just be the stringified JSON instead of calling `.stringify`? Feels a little less unit-test-ish to be calling function in the assertion, maybe? Maybe not? 😆